.github/workflows/distcheck.yaml

```diff
@@ -5,7 +5,25 @@ on:
       - master
       - main
       - v*
+    paths:
```

Comment:
"topology" should be in here too. Also, would it make more sense for this to be an exclude list, rather than an include list?

Reply:
Added topology. As far as inverting the list, I'm not sure how that will behave. You can invert a match [per the docs](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths) but suppose there's a pull request that makes both code changes and changes to jenkins ci definitions -- we want PR CI to run in that case. If we invert the match, does it still run?

I don't see any real reason to waste time figuring it out, this will do for now.